examples/tv-app/android/App/content-app/src/main/java/com/example/contentapp/AttributeHolder.java

```diff
@@ -1,13 +1,92 @@
 package com.example.contentapp;
 
+import android.util.Log;
 import com.matter.tv.app.api.Clusters;
 import java.util.HashMap;
 import java.util.Map;
 
 /** Class to hold attribute values to help test attribute read and subscribe use cases. */
 public class AttributeHolder {
-  private static AttributeHolder instance = new AttributeHolder();
   private Map<Long, Map<Long, Object>> attributeValues = new HashMap<>();
+  private static final String TAG = "AttributeHolder";
+
+  public static final String TL_LONG =
```

Comment:
There should be a comment saying what TL_LONG/SHORT/BAD are. what is a `TL` ?

Reply:
Added comment.